blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+---
+layout: post
+title: Predefined dialog in Blazor Dialog Component | Syncfusion
+description: Checkout and learn here all about Predefined dialog in Syncfusion Blazor Dialog component and much more.
```

Comment:
description: Check out and learn all about Predefined dialog in Syncfusion Blazor Dialog component and much more here.